exp-player/addon/components/exp-player.js

```diff
@@ -18,6 +18,7 @@ export default Ember.Component.extend(FullScreen, {
     conditions: null,
 
     frameIndex: 0, // Index of the currently active frame
+    framePage: 1, // Index of the currently visible page within a frame
```

Comment:
If we're going to add this to the exp-player we should unify how we're indexing `framePage` with `frameIndex` (both should be zero indexed)